core/container/src/main/java/org/wildfly/swarm/Swarm.java

```diff
@@ -123,8 +122,6 @@
 @Vetoed
 public class Swarm {
```

Comment:
Another possibility is to make Swarm implements SwarmInfo, then it will not break code that use `Swarm.VERSION`